src/main/java/uk/gov/ons/ssdc/caseprocessor/utils/Constants.java

```diff
@@ -1,5 +1,9 @@
 package uk.gov.ons.ssdc.caseprocessor.utils;
 
+import java.util.Set;
+
 public class Constants {
-  public static final String EVENT_SCHEMA_VERSION = "v0.3_RELEASE";
+  public static final String OUTBOUND_EVENT_SCHEMA_VERSION = "v0.3_RELEASE";
+  public static final Set<String> ALLOWED_INBOUND_EVENT_SCHEMA_VERSIONS =
+      Set.of("v0.3_RELEASE", "0.4.0-DRAFT", "0.4.0");
```

Comment:
Why aren't the two added versions in the same format? Happy to move to using 'proper' semantic versioning for this, just want to clarify.